src/core/ts/web-worker-scripts/translation-worker.js/index.ts

```diff
@@ -120,16 +162,34 @@ skip-cost: true
 cpu-threads: 0
 quiet: true
 quiet-translation: true
-shortlist:
-    - /${languagePair}/lex.${languagePair}.s2t
-    - 50
-    - 50
+gemm-precision: int8shift
 `;
 
-    console.log("modelConfig: ", modelConfig);
+    console.log(
+      "modelConfigWithoutModelAndShortList: ",
+      modelConfigWithoutModelAndShortList,
+    );
 
     // Instantiate the TranslationModel
-    model = new Module.TranslationModel(modelConfig);
+    const modelBuffer = downloadedModelFilesByType.model.arrayBuffer;
+    const shortListBuffer = downloadedModelFilesByType.lex.arrayBuffer;
+
+    console.debug({ modelBuffer, shortListBuffer });
```

Comment:
Btw, do you intend to print model and shortlist buffer on console? It might be too much. Right?

Reply:
The browser console is smart enough not to get spammed by this, but it is definitely unintentional to leave this console statement here and it will be cleaned up. :)